src/App.tsx

```diff
@@ -34,62 +35,118 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((goodsA: string, goodsB: string) => {
```

Comment:
You should give the variables proper names - they should describe what is stored inside. It shouldn't be in plural too in that case
```suggestion
  visibleGoods.sort((currentGood: string, nextGood: string) => {
```